### ISSUE TITLE
FMA links in Protégé do not direct to referenced term

### ISSUE THREAD

**Issue by bvarner-ebi** [obophenotype/uberon#2923]
Clicking FMA hyperlinks in Protégé direct to URL 
of the format https://bioportal.bioontology.org/virtual/2006/FMA:#####
with the message "The page you are looking for wasn't found. Please try again."

However, clicking the FMA links in OLS will direct to the FMA class in OLS. The PURL for the FMA class resolves to an Ontobee page, as expected.

For example, [UBERON:0002079 'left cardiac atrium'](https://www.ebi.ac.uk/ols4/ontologies/uberon/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FUBERON_0002079) has dbxref annotation FMA:7097. 
- In Protégé, clicking the FMA hyperlink directs to page https://bioportal.bioontology.org/virtual/2006/FMA:7097 with message "The page you are looking for wasn't found. Please try again."
- In OLS, clicking the dbxref link with the FMA directs here: https://www.ebi.ac.uk/ols4/ontologies/fma/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FFMA_7097?lang=en
- In OLS, the FMA PURL is http://purl.obolibrary.org/obo/FMA_7097 and resolves to: https://ontobee.org/ontology/FMA?iri=http://purl.obolibrary.org/obo/FMA_7097

Should Uberon FMA links in Protégé resolve to Ontobee or some other valid page instead of a page that states "The page you are looking for wasn't found. Please try again."?

**Comment by cmungall** [comment]
I don't think this is an uberon issue (although we should add prefix expansions as soon as this is supported in the owlapi obo writer)

I don't know how Protege decides to resolve xrefs, I would hope it uses something like bioregistry

note there is a related issue that there are effectively two versions of FMA, OBO purls resolve to a very old one
https://github.com/OBOFoundry/OBOFoundry.github.io/issues/21

But I don't think any of this is an uberon issue..

**Comment by gouttegd** [comment]
Protégé uses several “[link extractors](https://github.com/protegeproject/protege/blob/648e200c104a9b0596849041a16ea86ebe82c06f/protege-editor-owl/src/main/java/org/protege/editor/owl/ui/renderer/OWLAnnotationCellRenderer2.java#L55)” to transform identifiers into clickable links, basically trying each extractor in turn until one returns a link.

Here, the problem seems to be that the resolver based on [identifiers.org](https://identifiers.org/) is tried before the OBO Foundry extractor, and it’s identifiers.org that [returns an incorrect URI for a FMA entry](https://registry.identifiers.org/registry/fma).

Bottom line is that it is primarily a identifiers.org issue (out-of-date data about FMA?), and secondarily a Protégé issue (maybe Protégé should no longer uses identifiers.org?). But in any case there’s nothing to do about it in Uberon.

**Comment by bvarner-ebi** [comment]
Thanks for the feedback, @cmungall, @gouttegd.
Do you recommend moving this to another repo?
